website/docs/api/tokenizer.md

```diff
@@ -15,11 +15,15 @@ Create a `Tokenizer`, to create `Doc` objects given unicode text.
 >
 > ```python
 > # Construction 1
+> import spacy
 > from spacy.tokenizer import Tokenizer
+> nlp = spacy.load("en_core_web_sm")
```

Comment:
I probably wouldn't load a morel here. The API docs are really supposed to show the more abstract construction and API instead of very specific usage examples (like the [usage docs](https://spacy.io/usage/linguistic-features#tokenization)).

The main message here should be that the `Tokenizer` takes an instance of `Vocab`, and that `create_tokenizer` takes an instance of `Language`. So the most abstract case would be this one:

```python
from spacy.vocab import Vocab
from spacy.tokenizer import Tokenizer
tokenizer = Tokenizer(Vocab())
```

But since this is really far removed from actual usage, we made the example use a generic `nlp` object to stand for "whichever `nlp` object you've created and are using". But I agree that it's nicer if you can actually copy-paste the example. So I'd suggest using the `English` class instead:

```python
from spacy.lang.en import English
from spacy.tokenizer import Tokenizer

nlp = English()
tokenizer = Tokenizer(nlp.vocab)
```